react/components/AutocompleteInput/SearchInput/index.tsx

```diff
@@ -24,14 +24,20 @@ const propTypes = {
   onBlur: PropTypes.func,
   /** Determine if the input and the button should be disabled */
   disabled: PropTypes.bool,
+  /** Determine the search bar size */
+  size: PropTypes.string,
```

Comment:
How about...

```suggestion
  size: PropTypes.oneOf(['small', 'regular', 'large']),
```

Reply:
I prefer the name `medium` instead of `regular` because it's more clarifier :thinking: what exactly is regular?

Reply:
`regular` seems to be used all across the components to define the "medium" size 🤔  Using `medium` instead would differ from the rest of them